packages/compass-aggregations/src/modules/aggregation.ts

```diff
@@ -225,11 +242,22 @@ const fetchAggregationData = (page: number): ThunkAction<
         });
       }
     } catch (e) {
-      dispatch({
-        type: ActionTypes.AggregationFailed,
-        error: (e as Error).message,
-      });
-      log.warn(mongoLogId(1001000106), 'Aggregations', 'Failed to run aggregation');
+      // On cancel, we show the previous state
+      if ((e as Error).name === PROMISE_CANCELLED_ERROR) {
+        dispatch({
+          type: ActionTypes.AggregationFinished,
+          documents: _documents,
+          page: _page,
+          isLast: _isLast,
+        });
```

Comment:
Just want to double check something: in CRUD view when cancelling we should the error banner saying that operation was cancelled and no documents, I am not sure that this current behavior was specifically discussed as the one we want and it's fine to keep it as is right now, but as this is yet again something that is different from CRUD view for seemingly no reason I think it might make sense to raise this and align the behaviors to whatever is the preferred one

Reply:
Yeah sounds good! I'll discuss these things with Claudia once she's back in the office.